### DISCUSSION TITLE
Spam emails

### DISCUSSION THREAD

**Discussion by DaviidM** [kaorahi/lizgoban#84] [General]
Is anybody else getting spam emails (such as attached)?
![OUTLOOK_4wpqz662DN](https://user-images.githubusercontent.com/66852680/209906115-895fe31d-45e1-454a-99d6-fe5be4630f4e.png)

**Comment by kaorahi** [comment]
I received such emails twice today. It seems that spams were posted to #81 and they were deleted soon. You can turn off the notification emails on #81 if you like ("Unsubscribe" button under "Notifications" label in the right side pane).

**Reply by DaviidM** [reply]
Sounds like a good idea - except I justgot another one... Re: [kaorahi/lizgoban] Spam emails (Discussion #84).
